bin/pycbc_splitbank

```diff
@@ -157,7 +144,7 @@ for num, (idx1, idx2) in enumerate(zip(index_list[:-1], index_list[1:])):
                     version=version.version, cvs_repository=version.git_branch,
                     cvs_entry_time=version.date).process_id
 
-    sngl_inspiral_table = lsctables.New(tabletype,columns=template_bank_table.columnnames)
```

Comment:
Actually, Tito, looking again, is the problem here? You have removed the columns line, which I think you want.

Reply:
Unfortunately, this is what fixes the problem reported by @MariaAssiduo!